src/test/javascript/spec/component/exam/participate/exam-participation.component.spec.ts

```diff
@@ -351,6 +351,12 @@ describe('ExamParticipationComponent', () => {
         expect(secondSubmission.isSynced).toBeTrue();
         expect(secondSubmission.submitted).toBeFalse();
 
+        if (!!comp.testRunId || comp.testExam) {
```

Comment:
Should you rely on the values in the tested component to determine the correct assertion?
Could you use the parameter of this method instead?

Reply:
Fixed, thanks!